src/main/java/com/spartronics4915/frc2019/auto/actions/CollectAccelerationData.java

```diff
@@ -47,13 +52,14 @@ public void start()
         mDrive.setOpenLoop(new DriveSignal((mReverse ? -1.0 : 1.0) * kPower, (mReverse ? -1.0 : 1.0) * (mTurn ? -1.0 : 1.0) * kPower));
         mStartTime = Timer.getFPGATimestamp();
         mPrevTime = mStartTime;
+        Logger.debug("Collecting acceleration data");
     }
 
     @Override
     public void update()
     {
         double currentVelocity =
-                (Math.abs(mDrive.getLeftVelocityTicksPer100ms()) + Math.abs(mDrive.getRightVelocityTicksPer100ms())) / Constants.kDriveEncoderPPR * Math.PI * 10;
+                mSide.getVelocitiesFromDrive(mDrive) / Constants.kDriveEncoderPPR * Math.PI * 10;
```

Comment:
need comment on units here

Reply:
also: it appears that side to characterize is currently unimplemented, right?

Reply:
hm... just now getting my head around SideToCharacterize...

Reply:
Yes. Also, the issue with the mysterious lack of dividing by 2 (and the ` * 2` [here](https://github.com/Spartronics4915/Spartronics-254Base2018/blob/b7cedbd9aa894ebc3ec1ffee0d8ce155012e78d5/src/main/java/com/spartronics4915/frc2019/planners/DriveMotionPlanner.java#L61)) have not been fully dealt with here.